onprem-environment

```diff
@@ -1,3 +1,5 @@
+UI_PORT=8080
+INVENTORY_PORT=8090
```

Comment:
What is inventory port in this context? Should this be called API or SERVICE_API port instead?